pkg/tracker/runner.go

```diff
@@ -31,16 +31,15 @@ func NewRunner(client rpc.ETHClient, db db.DB) (*Runner, error) {
 }
 
 // Start will kick off the runner until the given exit channel selects.
-func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
+func (r *Runner) Start(ctx context.Context, logger log.Logger, exitCh chan int) error {
```

Comment:
We should add a constructor here as well and add the logger to the instructor like in all the other changes.
```suggestion
func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
```